.vsts.pipelines/phases/ci-linux.yml

```diff
@@ -1,20 +1,16 @@
 parameters:
-  dockerRegistryPassword: ''
-  dockerRegistryServer: ''
-  dockerRegistryUserName: ''
-  matrix: null
-  name: ci
-  queueDemands: Agent.OS -equals Linux
-  queueName: DotNet-Build
+  imageName: null
+  matrix:
+    Production: {}
+  phase: null
+  queueDemands: docker
+  queueName: dnceng-linux-external-temp
 
 phases:
-- phase: ${{ parameters.name }}
+- phase: ${{ parameters.phase }}
   variables:
-    # Prefix to distinguish artifacts from different legs. No documented variable for this exists.
-    artifactName: no_artifact_name
-    buildLoggingOptions: ''
-    buildConfiguration: Release
-    buildOfflineTarball: false
+    # Prefix to distinguish artifacts from different legs.
+    artifactName: ${{ format('$(type)_{0}', parameters.phase) }}
```

Comment:
Why an underscore separator here when the type param contains space separators anyway?

Reply:
This is probably leftover from an older understanding of AzDO's naming requirements--changing to space.